sysdig/provider.go

```diff
@@ -43,6 +43,7 @@ func Provider() terraform.ResourceProvider {
 			"sysdig_secure_rule_process":         resourceSysdigSecureRuleProcess(),
 			"sysdig_secure_rule_syscall":         resourceSysdigSecureRuleSyscall(),
 			"sysdig_secure_rule_falco":           resourceSysdigSecureRuleFalco(),
+			"sysdig_user":                        resourceSysdigUsers(),
```

Comment:
Please, rename to `resourceSysdigUser()` to follow existing conventions

Reply:
Thank you for your review!
I fixed it. and I also changed `users` to `user` on `models.go`
https://github.com/draios/terraform-provider-sysdig/pull/12/commits/7f70b61d3647a80a1e498b248a1242ef7b841394#diff-e9b4a38ce91c5f14f45388bf7c86eb4aR213